test/unit/components/messages/message.test.js

```diff
@@ -0,0 +1,61 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var expect = chai.expect;
+
+var Message = require('../../../../app/components/messages/message');
+
+describe('Message', function () {
+  describe('getInitialState', function() {
+    it('should return an object with editing set to false', function() {
+      console.warn = sinon.spy();
+      // TODO: consider requiring Proptype: theNote (maybe define as a shape as well?)
+      //   -> Properties of theNote are accessed in componentDidMount()
```

Comment:
The way it is currently coded I don't believe imageSize or onSaveEdit are required, but I will make the update for theNote